### ISSUE TITLE
Field 'draft' doesn't exist on type 'MergeRequest'

### ISSUE THREAD

**Issue by bo1ogov** [TeamCodeStream/codestream#765]
Hi! Not too long ago I already had a similar problem with the 'reviewers' field (https://github.com/TeamCodeStream/codestream/issues/748)

https://user-images.githubusercontent.com/74357329/141728230-dc0803dc-dc41-4475-9241-c212307f6b8c.mp4

Error message:
```
Error Loading Pull Request:
Field 'draft' doesn't exist on type 'MergeRequest': {"response":{"errors":[{"message":"Field 'draft' doesn't exist on type 'MergeRequest'","locations":[{"line":25,"column":7}],"path":["query GetPullRequest","project","mergeRequest","draft"],"extensions":{"code":"undefinedField","typeName":"MergeRequest","fieldName":"draft"}},{"message":"Field 'reviewers' doesn't exist on type 'MergeRequest'","locations":[{"line":65,"column":7}],"path":["query GetPullRequest","project","mergeRequest","reviewers"],"extensions":{"code":"undefinedField","typeName":"MergeRequest","fieldName":"reviewers"}}],"status":200,"headers":{}},"request":{"query":"query GetPullRequest($fullPath: ID!, $iid: String!, $after: String) { project(fullPath: $fullPath) { name mergeRequestsEnabled mergeRequestsFfOnlyEnabled removeSourceBranchAfterMerge mergeRequest(iid: $iid) { approvedBy(first: 50) { nodes { avatarUrl name login: username } } id iid createdAt sourceBranch targetBranch title description webUrl state mergedAt draft reference projectId mergeWhenPipelineSucceeds mergeableDiscussionsState author { name login: username avatarUrl } diffRefs { baseSha headSha startSha } commitCount project { name path webUrl fullPath } sourceProject { name path webUrl fullPath } upvotes downvotes milestone { title id webPath dueDate } subscribed userDiscussionsCount discussionLocked forceRemoveSourceBranch reviewers { nodes { id name login: username avatarUrl } } timeEstimate totalTimeSpent updatedAt userPermissions { adminMergeRequest canMerge } discussions(first: 50, after: $after) { pageInfo { endCursor hasNextPage } nodes { ...discussionFragment } } } }}fragment discussionFragment on Discussion { createdAt id notes { nodes { author { name login: username avatarUrl } body bodyHtml createdAt discussion { id replyId createdAt } id position { x y newLine newPath oldLine oldPath filePath diffRefs { baseSha headSha startSha } } project { name } resolvable resolved resolvedAt resolvedBy { login: username avatarUrl } system systemNoteIconName updatedAt userPermissions { adminNote readNote resolveNote awardEmoji createNote } } } replyId resolvable resolved resolvedAt resolvedBy { login: username avatarUrl }}","variables":{"fullPath":"dev/Core","iid":"7064"}}}
```

GLSM Version: 
```json
{
    "version": "13.6.1",
    "revision": "1b6a590b197"
}
```

Rider Info:
```
JetBrains Rider 2021.2.2
Build #RD-212.5284.64, built on October 9, 2021
Runtime version: 11.0.12+7-b1504.28 amd64
Windows 10 10.0
```

CodeStream Version:
```
12.1.0+206
```

Logs sent to support@codestream.com

**Comment by bcanzanella** [comment]
thank you for the report @bo1ogov and sorry you're hitting a snag. we're checking it out.

**Comment by planteater** [comment]
@bo1ogov - This should be fixed in v12.2.
